src/model/queryable.ts

```diff
@@ -25,6 +25,12 @@ export interface BaseQueryOptions {
    * that are referenced from within the package.
    */
   externalPackages?: boolean;
+
+  /**
+   * Do not include any features that are only reachable via paths that include
+   * lazy import edges.
+   */
+  strictImports?: boolean;
```

Comment:
that's certainly a bit simpler, but the name here doesn't have a clear meaning to me. How about a convention of a "no" prefix for options we'd rather have a default of true. So `noLazyImports`?